lib/chat_web/live/main_live/index.html.heex

```diff
@@ -500,12 +500,16 @@
                     uploads={@uploads_metadata}
                   />
 
-                  <Layout.Uploader.mobile_uploader
-                    config={@uploads.file}
-                    operating_system={@operating_system}
-                    type="dialog"
-                    uploads={@uploads_metadata}
-                  />
+                  <div class="max-h-[280px] bottom-16 fixed w-full overflow-y-scroll flex flex-col-reverse">
```

Comment:
The component is also used in the rooms.
Can you maybe put this code into the component so that both dialog and room contain this fix?

Reply:
fixed